packages/L10n/stories/examples/FakeAppWithLocales/Locales/en.js

```diff
@@ -0,0 +1,9 @@
+const locales = {
+  en: {
+    translation: {
+      help_text: "Uh-oh, there was an error! <0>Go to tab 2</0>. <1>Go to Google</1>.",
+      help_text_plural: "Uh-oh, there were {{count}} errors! <0>Go to tab 2</0>. <1>Go to Google</1>.",
+    },
+  },
+};
+export default locales;
```

Comment:
`0` and `1` are the indexes of the child components that get passed into `<Trans>`